lib/sign_in/idme/service.rb

```diff
@@ -49,12 +49,18 @@ def user_info(token)
       private
 
       def get_public_jwks
-        unless config.public_jwks
-          response = perform(:get, config.public_jwks_path, nil, nil)
-          config.public_jwks = parse_public_jwks(response:)
-          Rails.logger.info('[SignIn][Idme][Service] Get Public JWKs Success')
+        return config.public_jwks if config.public_jwks
```

Comment:
If we're going to cache this, why not just remove `public_jwks` attribute from the Configuration object?

Reply:
from a perf perspective I don't mind the extra redis hit

Reply:
Ah yeah, good call